hoff.nix

```diff
@@ -0,0 +1,86 @@
+{ lib, haskellPackages, nix-gitignore, git, coreutils, openssh }:
+  haskellPackages.mkDerivation {
+    pname = "hoff";
+    version = "0.25";
+
+    src =
+      let
+        # We do not want to include all files, because that leads to a lot of things that nix
+        # has to copy to the temporary build directory that we don't want to have in there
+        # (e.g. the `.stack-work` directory, the `.git` directory, etc.)
+        prefixWhitelist = builtins.map builtins.toString [
+          ./app
+          ./package
+          ./src
+          ./static
+          ./tests
+          ./tools
```

Comment:
Same here, as far as I can tell `send-webhook.py` (the only file in here) isn't used in the build.

Reply:
Good catch